_includes/story.md

```diff
@@ -781,44 +791,50 @@ Let's walk through this algorithm in the context of our example:
 - then, we provision `Queue 2` with: `2,000 / 2,516.19 ~ 1`
 - we have now provisioned the queues with all 25 workers
 
+{: .center}
 ![ASP_diagram_2](/img/ASP_diagram_2.png){:width="350"}
 
-Next, we will start looking at how, as BJP builders, we can help a user make better decision about his/her background job system.
+Next, we will start looking at how, as BJP developers, we can help user make a better decision about her background job system.
 
 ## Reporting
 
-Reporting is an important feature to implement when building a BJP. It allows developers to gain insight into the overall state of jobs, the job types, jobs that failed, and how many jobs are completed over time, which the developer can use to make better decisions about adjustments needed to the background job system.
+Reporting is an important feature to have when building a BJP. It allows developers to gain insight into the overall state of jobs, their types, jobs that failed, and how many of them complete over time. This information can be used to make better decisions about adjustments needed to the background job system.
```

Comment:
I think `implement` was better than `have`:
> important feature to implement when building a BJP

using `implement` means that we're talking about a building standpoint. Have makes it sound like it's already there.